griptape/drivers/embedding/openai_embedding_driver.py

```diff
@@ -26,36 +23,26 @@ class OpenAiEmbeddingDriver(BaseEmbeddingDriver):
 
     model: str = field(default=DEFAULT_MODEL, kw_only=True)
     dimensions: int = field(default=DEFAULT_DIMENSIONS, kw_only=True)
-    api_type: str = field(default=openai.api_type, kw_only=True)
-    api_version: Optional[str] = field(default=openai.api_version, kw_only=True)
-    api_base: str = field(default=openai.api_base, kw_only=True)
-    api_key: Optional[str] = field(default=Factory(lambda: os.environ.get("OPENAI_API_KEY")), kw_only=True)
-    organization: Optional[str] = field(default=openai.organization, kw_only=True)
+    base_url: str = field(default=None, kw_only=True)
+    api_key: Optional[str] = field(default=None, kw_only=True)
+    organization: Optional[str] = field(default=None, kw_only=True)
+    client: openai.OpenAI = field(
+        init=False,
```

Comment:
Same question as in `AzureOpenAiEmbeddingDriver`.

Reply:
Removed